webapp/components/Paginate/Paginate.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+  <ds-flex direction="row-reverse">
+    <ds-flex-item width="50px">
+      <ds-button @click="next" :disabled="!hasNext" icon="arrow-right" primary />
+    </ds-flex-item>
+    <ds-flex-item width="50px">
+      <ds-button @click="back" :disabled="!hasPrevious" icon="arrow-left" primary />
```

Comment:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2019-10-30T08:27:13Z" title="Wednesday, October 30th 2019, 9:27:13 am +01:00">Oct 30, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,26 @@
+<template>
+  <ds-flex direction="row-reverse">
+    <ds-flex-item width="50px">
+      <ds-button @click="next" :disabled="!hasNext" icon="arrow-right" primary />
+    </ds-flex-item>
+    <ds-flex-item width="50px">
+      <ds-button @click="back" :disabled="!hasPrevious" icon="arrow-left" primary />
```


styling-wise I think these could just be regular buttons – without the `primary` attribute

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-10-30T14:40:17Z" title="Wednesday, October 30th 2019, 3:40:17 pm +01:00">Oct 30, 2019</time>_

---


I tried this out and you couldn't really see the buttons... maybe there is another idea besides just removing the `primary` attribute @alina-beck ??
![Screenshot from 2019-10-30 15-40-12](https://user-images.githubusercontent.com/26943915/67868010-909ec300-fb2b-11e9-8b92-9854ecfb9b46.png)